chains/evm/chain.go

```diff
@@ -55,6 +55,11 @@ func SetupDefaultEVMChain(rawConfig map[string]interface{}, txFabric calls.TxFab
 	t := signAndSend.NewSignAndSendTransactor(txFabric, gasPricer, client)
 	bridgeContract := bridge.NewBridgeContract(client, common.HexToAddress(config.Bridge), t)
 
+	isRegisteredRelayer, _ := bridgeContract.IsRelayer(common.HexToAddress(config.GeneralChainConfig.From))
```

Comment:
Why you are not returning error?

Reply:
I think it would be redundant, if an error is thrown in some underlying function it will be logged/thrown either way.
This is only an informational helper to make life easier while debugging.

Reply:
But log.Error does not stop function execution.So relayer will continue working